cves/kernel/CVE-2018-14734.yml

```diff
@@ -131,10 +129,15 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    There were no automated tests found in the surrounding directories and the
+    commit message and comments do not indicate that there were any tests involved.
+  fix: false
+  fix_answer: |
+    Automated tests were not involved and the fix was to separate the two functions.
+    Originally, the problematic variable was given an ID and published, but the ID
+    allocation was separated from the publish function.
```

Comment:
Again maybe add this description of the fix somewhere else as this is just asking if the fix contains tests